index.html

```diff
@@ -364,7 +384,7 @@
 		<h2>Pointer Event Types</h2>
 		<section>
 			<h3>Firing events using the <code>PointerEvent</code> interface</h3>
-			To <dfn>fire a pointer event name e</dfn> means to <dfn>fire an event named e</dfn> as defined in [[!DOM4]] with an event using the <a>PointerEvent</a> interface whose attributes are set as defined in <a href="#pointerevent-interface"><code>PointerEvent</code> Interface</a>.</p>
+			To <dfn>fire a pointer event name e</dfn> means to <dfn>fire an event named e</dfn> as defined in [[!DOM4]] with an event using the <a>PointerEvent</a> interface whose attributes are set as defined in <a href="#pointerevent-interface"><code>PointerEvent</code> Interface</a>.
```

Comment:
for consistency with previous sections, maybe wrap this in a `<div><p>...</p></div>` as well?